src/main/java/in/cognitron/lms/service/Hello.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2019 cognitron.in
+ * all rights reserved
+ */
+
+package in.cognitron.lms.service;
+
+import org.simplity.fm.core.Message;
+import org.simplity.fm.core.MessageType;
+import org.simplity.fm.core.serialize.IInputObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import in.cognitron.lms.LmsService;
+import in.cognitron.lms.LmsServiceContext;
+
+public class Hello extends LmsService {
+	protected static final Logger logger = LoggerFactory.getLogger(Hello.class);
+	private static final String SERVICE_NAME = "Hello";
+	@Override
+	public String getId() {
+		return SERVICE_NAME;
+	}
+
+	@Override
+	public void execute(final LmsServiceContext ctx, final IInputObject inputObject) throws Exception {
+				ctx.addMessage(Message.newMessage(MessageType.Info,"Hello From the other side"));
```

Comment:
Why is this line indented so far to the right?  Is your linter not picking this up?